modules/Microsoft.DesktopVirtualization/workspaces/.test/common/deploy.test.bicep

```diff
@@ -81,7 +81,16 @@ module testDeployment '../../deploy.bicep' = {
         principalType: 'ServicePrincipal'
       }
     ]
+<<<<<<< HEAD
+    workspaceDescription: 'This is my first AVD Workspace'
```

Comment:
We have twice description and friendly name, the outdated ones should be removed

Reply:
workspaceFriendlyName removed